Source/JavaScriptCore/API/JSValue.h

```diff
@@ -234,13 +234,13 @@ NS_CLASS_AVAILABLE(10_9, 7_0)
  into a JavaScript value according to the above conversion.
 */
 /*!
-@method
+@property
 @abstract Convert this JSValue to an Objective-C object.
-@discussion The JSValue is converted to an Objective-C object according 
+@discussion The JSValue is converted to an Objective-C object according
  to the conversion rules specified above.
 @result The Objective-C representation of this JSValue.
 */
-- (id)toObject;
+@property (nonatomic, readonly, strong) id toObject;
```

Comment:
Here's an example of something I think the migrator does wrong. This is a _conversion_, not an accessor. I don't think it's appropriate to expose it as a getter property, because that hides that the implementation is doing nontrivial work. i.e. in Swift, it _should_ be `JSValue.toObject()`, not `getter:JSValue.toObject`.

Reply:
Fixed!